src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTests.cs

```diff
@@ -137,12 +137,12 @@ static void Main(string[] args)
     static void Main(string[] args)
     {
         using (null)
-        using (null)$$
+        using (null)
     }
 }
 ";
 
-            AssertFormatWithPasteOrReturn(expected, code, allowDocumentChanges: true, isPaste: false);
+            AssertFormatAfterTypeChar(code, expected);
```

Comment:
❗️ This test is no longer correct. It should be updated in two ways:

1. It should be renamed to indicate that it is verifying that using statements _are not_ formatted on return
2. It should be updated to verify (1)

If there is no separate test verifying that typing `)` triggers the completion, one could be added as a new test next to this one.

Reply:
Sure.